python/tk_multi_publish2/processing/task.py

```diff
@@ -60,6 +61,16 @@ def create_task(cls, plugin, item, is_visible, is_enabled, is_checked):
         logger.debug("Created %s" % task)
         return task
 
+    def is_same_task_type(self, other_task):
+        """
+        Indicates if this plugin instance wraps the same plugin type as another
+        plugin instance.
+
+        :param other_task: The other plugin to test against.
+        :type other_task: :class:`Task`
+        """
+        return self._plugin.name == other_task._plugin.name
```

Comment:
I don't think this is sufficient. We have totally different plugins that each use **Publish to Shotgun** for the name. May need to check their source file and inheritance chain??? 😬

Reply:
You are confusing the `Plugin.name`, which returns the name of the plugin in the environment and `Plugin.plugin_name`, which returns the name of the plugin as returned by the `plugin_name` property in the hook instance.

I do agree however that even `name` could be confusing, so I should actually be comparing plugin instances, as each is loaded only one and two same plugin instance have therefore the same UI.